public/style.css

```
@@ -102,6 +102,11 @@ button:hover {
     vertical-align: middle;
     width: 100%;
 }
+.input-outer-wrapper.small{
```

Minor spacing inconsistency:

```suggestion

.input-outer-wrapper.small{
```